catalog/tests/dags/maintenance/test_pr_review_reminders.py

```diff
@@ -315,30 +427,104 @@ def test_does_not_ping_if_pr_has_min_required_approvals(github, urgency, base_br
     )
 
     github["pulls"] += [past_due_pull]
-    github["pull_reviews"][past_due_pull["id"]] = [
+    github["pull_reviews"][past_due_pull["number"]] = [
         make_review("APPROVED"),
     ] * min_required_approvals
+    github["events"][past_due_pull["number"]] = make_urgent_events(urgency, events)
 
     post_reminders("not_set", dry_run=False)
 
     assert past_due_pull["number"] not in github["posted_comments"]
 
 
 @parametrize_urgency
-def test_does_not_ping_if_no_reviewers(github, urgency):
-    past_due_pull = make_pull(urgency, past_due=True)
+@parametrize_possible_pingable_events
+def test_does_not_ping_if_no_reviewers(github, urgency, events):
+    past_due_pull = make_pull(urgency, old=True)
     past_due_pull["requested_reviewers"] = []
-    not_due_pull = make_pull(urgency, past_due=False)
+    not_due_pull = make_pull(urgency, old=False)
     not_due_pull["requested_reviewers"] = []
-    github["pulls"] += [past_due_pull, not_due_pull]
+    old_but_not_due_pull = make_pull(urgency, old=True)
+
+    github["pulls"] += [past_due_pull, not_due_pull, old_but_not_due_pull]
     github["pull_comments"][past_due_pull["number"]].append(
         make_pr_comment(is_reminder=False)
     )
 
+    github["events"][past_due_pull["number"]] = make_urgent_events(urgency, events)
+    github["events"][not_due_pull["number"]] = make_non_urgent_events(events)
+    github["events"][
+        old_but_not_due_pull["number"]
+    ] = make_non_urgent_reviewable_events(events)
+
     for pr in [past_due_pull, not_due_pull]:
         _setup_branch_protection(github, pr)
 
     post_reminders("not_set", dry_run=False)
 
     assert past_due_pull["number"] not in github["posted_comments"]
     assert not_due_pull["number"] not in github["posted_comments"]
+    assert old_but_not_due_pull["number"] not in github["posted_comments"]
+
+
+@parametrize_urgency
+@pytest.mark.parametrize(
+    "events",
```

Comment:
We can't because the test case is only relevant for PRs that have a ready for review event. `possible_pingable_events` includes the one case where there wouldn't be a ready for review event on a PR that could be eligible: a PR that was never drafted. I'll add a comment explaining that to the code.